v0/destinations/ga4/transform.js

```diff
@@ -242,11 +244,27 @@ const responseBuilder = (message, { Config }) => {
     payload.params = getGA4CustomParameters(
       message,
       ["properties"],
-      GA4_RESERVED_PARAMETER_EXCLUSION,
+      GA4_RESERVED_PARAMETER_EXCLUSION.concat(GA4_PARAMETERS_EXCLUSION),
       payload
     );
   }
 
+  // take optional engagement_time_msec, session_id parameters for reports
+  if (message.type === "identify" || message.type === "group") {
```

Comment:
Why can t we put this in the mapping json itself rather than writing extra code to set this?